vimrc

```diff
@@ -202,12 +202,10 @@ let g:rails_projections = {
       \   }
       \ }
 
-if exists(':RainbowParenthesesToggle')
```

Comment:
This condition was prohibiting rainbow parentheses from working under any circumstances.  It is not clear to me that it adds any value any more, but please lmk if you feel otherwise.